web/api/v1/api.go

```diff
@@ -1204,6 +1211,10 @@ func (api *API) metricMetadata(r *http.Request) apiFuncResult {
 					m := metadata{Type: mm.Type, Help: mm.Help, Unit: mm.Unit}
 					ms, ok := metrics[mm.Metric]
 
+					if len(metrics[mm.Metric]) >= limitPerMetric {
```

Comment:
Any performance concerns with calling `len()` in this loop?

Reply:
Do we have benchmark requirements that we could test against?

I'm not really sure how to implement it without using `len` inside this for though 🤔